config/config.json

```diff
@@ -15,7 +15,9 @@
         "SessionLengthWebInDays": 30,
         "SessionLengthMobileInDays": 30,
         "SessionLengthSSOInDays": 30,
-        "SessionCacheInMinutes": 10
+        "SessionCacheInMinutes": 10,
```

Comment:
@enahum shouldn't be any changes to config.json, correct?

We have a separate tix to fix this problem with the build environment changing config.json and versiong.go.